DPRHTML/search.html

```diff
@@ -52,14 +52,8 @@
     }
     $(document).ready(function() {
       setSearchParams();
-      $('#isearchInfo')
-        .each(function() {
-          $(this).popover({
-            trigger: 'click',
-            html : true,
-            content: () => $('#isearch-popover-content').html(),
-          })
-        });
+
+      DPR_PAL.enablePopover('#isearchInfo', 'click');
```

Comment:
Here I also would prefer trigger _hover_ instead of _click_.

Reply:
done